roles/repository/molecule/default/molecule.yml

```diff
@@ -4,7 +4,17 @@ dependency:
 driver:
   name: docker
 platforms:
-  - name: repository
+  - name: dummy
```

Comment:
this is for having hostvars for these hosts available without really having to provision them, correct?

Reply:
not really the hosts vars but more the structure of the inventory which we require. Basically I think we should do that everywhere in molecule config rather than linking the inventory file in ansible args